### PR TITLE
Update to marketplace-core-modules 2.0.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @mstriemer

### CODE REVIEW

File: src/media/js/main.js [hunk 1]
```diff
@@ -1,59 +1,45 @@
...
-        'z'
-    ],
-function() {
-    var console = require('log')('main');
-    var z = require('z');
-
-    console.log('Dependencies resolved, starting init');
-
-    z.body.addClass('html-' + require('l10n').getDirection());
+define('main', ['init'], function() {
```
Comment (mstriemer) [inline src/media/js/main.js]: Fireplace depends on `settings_app` and `routes` here now. You might want to do that also.

File: src/media/js/routes.js [hunk 2]
```diff
@@ -1,29 +1,25 @@
...
         });
     });
-    return routes;
-});
 
-})();
+    router.addRoutes(routes);
+
+    router.api.addRoutes({
+        'hello-names': '/api/v2/hello/names/',
```
Comment (mstriemer) [inline src/media/js/routes.js]: Is this used or was this an example?

**Comment by mstriemer** [comment]
The changes look good. Should be some updates the bower.json and package.json files though. I didn't have time to run this locally but I can try it out tomorrow to be sure.

**Comment by mstriemer** [comment]
I see you're importing `*` as the version for our packages. I'm not sure how updating works for that but that isn't a big deal for something that's probably going to change a bunch.

I'm assuming you need to update marketplace-gulp for stylus? I had to manually compile `index.styl` with a recent stylus to get jeet to compile since they're using a hash object in [`_settings.styl`](https://github.com/mojotech/jeet/blob/13f817dbbad5310a3bea00471f521bcce53182d4/stylus/jeet/_settings.styl).

Seems to work other than that. r+wc

**Comment by chuckharmston** [comment]
I plan on pinning to versions once development on this slows down a bit.

# TODO
- [ ] Ship a version of marketplace-gulp with a new version of Stylus.

**Comment by mstriemer** [comment]
I'm not sure how well a new stylus in marketplace-gulp will go over. I know fireplace had some problems with it. It would be nice to have different stylus versions in different projects but I'm not sure how to do that. A more recent stylus everywhere would definitely be the best though.

**Comment by chuckharmston** [comment]
Yeah, I already pinged @ngokevin about possible approaches. It might be a new major version + maintaining two separate release trees until each project gets a chance to update. I don't think that'd be too painful.

I haven't been able to find release notes for Stylus, so assessing backwards-compatability is tricky.

**Comment by chuckharmston** [comment]
Thanks @mstriemer!

**Pull request merged**